weld/llvm.rs

```diff
@@ -85,7 +85,7 @@ pub fn compile_program(program: &Program, opt_passes: &Vec<Pass>, llvm_opt_level
 
     gen.add_function_on_pointers("run", &sir_prog)?;
     let llvm_code = gen.result();
-    trace!("LLVM program:\n{}\n", &llvm_code);
+    debug!("LLVM program:\n{}\n", &llvm_code);
```

Comment:
Make this trace again - the trace stuff will be easier to see soon in the repl since you can pass debug level as a command line flag